### PR TITLE
fix: search also needs a feed id

### PR DESCRIPTION
**Pull request by rebelchris**
We need to add feed id to search as well.
(excluded specific tests)

### CODE REVIEW

File: src/schema/search.ts [hunk 1]
```diff
@@ -483,6 +519,9 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
                 })
                 .andWhere('cp."userId" = :cpUserId', {
                   cpUserId: ctx.userId,
+                })
+                .andWhere('cp."feedId" = :cpFeedId', {
```
Comment (capJavert) [inline src/schema/search.ts:523]: Not 100%, but can you check that user can't just send someone else feedId and search through it, I think above `userId` condition should make it impossible, but just to be sure. Adding a test for that would be golden!

Comment (capJavert) [reply src/schema/search.ts:523]: Same for source search

Comment (rebelchris) [reply src/schema/search.ts:523]: Ah good point, let me add it and validate :)

**Review by capJavert — APPROVED** [review]
Tnx for checking and adding tests!

**Pull request merged**